src/cata_imgui.cpp

```diff
@@ -0,0 +1,823 @@
+#include "cata_imgui.h"
+#include <stack>
+#include "output.h"
+#include "ui_manager.h"
+#include "color.h"
+#include "input.h"
+#include <type_traits>
+#include <imgui/imgui.h>
+#include <imgui/imgui_internal.h>
+#if !(defined(TILES) || defined(WIN32))
+#include <curses.h>
+#include <imtui/imtui-impl-ncurses.h>
+#include <imtui/imtui-impl-text.h>
+#include "color_loader.h"
```

Comment:
Sort and group includes (self header, std library, externally managed third party, in-repo third party, cdda headers)
```suggestion
#include "cata_imgui.h"

#include <stack>
#include <type_traits>

#include <imgui/imgui.h>
#include <imgui/imgui_internal.h>

#include "color.h"
#include "input.h"
#include "output.h"
#include "ui_manager.h"

#if !(defined(TILES) || defined(WIN32))
#include <curses.h>
#include <imtui/imtui-impl-ncurses.h>
#include <imtui/imtui-impl-text.h>

#include "color_loader.h"
```

Reply:
sure 👍